langchain4j-ollama/src/main/java/dev/langchain4j/model/ollama/OllamaClient.java

```diff
@@ -37,6 +37,8 @@ public OllamaClient(String baseUrl, Duration timeout) {
                 .connectTimeout(timeout)
                 .readTimeout(timeout)
                 .writeTimeout(timeout)
+                .addInterceptor(new OllamaRequestLoggingInterceptor())
```

Comment:
Could you please make it configurable via builder of every `Ollama*Model` like in OpenAI?
```java
OllamaChatModel.builder()
                ...
                .logRequests(true)
                .logResponses(true)
                ...
```

Reply:
Thanks for review, I have added the code you mentioned.